web/packages/teleport/src/TopBar/TopBar.tsx

```diff
@@ -191,29 +173,37 @@ export function TopBar({ CustomLogo }: TopBarProps) {
                 </NavigationButton>
               );
             })}
-          </>
-        )}
+          </Flex>
+        </>
+      )}
+      {feature?.hideFromNavigation && (
+        <ButtonIconContainer onClick={handleBack}>
+          <ArrowLeft size="medium" />
+        </ButtonIconContainer>
+      )}
+      <Flex height="100%" alignItems="center">
         {!hasDockedElement && assistEnabled && (
           <ButtonIconContainer onClick={() => setShowAssist(true)}>
             <BrainIcon />
           </ButtonIconContainer>
         )}
         <Notifications />
         <UserMenuNav username={ctx.storeUser.state.username} />
+        {showAssist && (
+          <Suspense fallback={null}>
+            <Assist onClose={() => setShowAssist(false)} />
```

Comment:
oops, did we test out if assist looks ok? (i forgot about it)

Reply:
Pic of docked assist (the rest of the assit view modes overlay ontop and dont affect the width)
<img width="2037" alt="Screenshot 2024-01-12 at 5 09 36 PM" src="https://github.com/gravitational/teleport/assets/5201977/1032d855-8890-4bf2-9d97-140545efba82">

Reply:
and the overlay  (cant really see much cuz its just on top but just showing the zindex wrking correctly)
<img width="1268" alt="Screenshot 2024-01-12 at 5 10 24 PM" src="https://github.com/gravitational/teleport/assets/5201977/e47df537-e6c1-4767-9448-916021b51028">

Reply:
thanks for the check 👍